AppServer/google/appengine/tools/devappserver2/module.py

```diff
@@ -32,6 +33,7 @@
 import threading
 import time
 import urllib
+import urllib2
```

Comment:
As far as I know urllib2 doesn't manage connections pool, so it opens creates new connection for every request. Shouldn't we try using `requests` or other lib which manager connection pool?

Reply:
That's a good point. We are making a lot of requests to the same socket.